robot_calibration/src/finders/checkerboard_finder.cpp

```diff
@@ -68,12 +72,14 @@ bool CheckerboardFinder::init(const std::string& name,
   // Publish where checkerboard points were seen
   publisher_ = nh.advertise<sensor_msgs::PointCloud2>(getName() + "_points", 10);
 
+  ROS_INFO_STREAM("points_x: " << points_x_ << "\npoints_y: " << points_y_ << "\nsize: " << square_size_ << "\nframe_id: " << frame_id_<< "\ncamera_sensor_name_: " << camera_sensor_name_ << "\nchain_sensor_name: " << chain_sensor_name_);
```

Comment:
might be worth  adding the getName() value here, since you might have multiple checkerboard finders.